src/uwtools/resources/rocoto.jsonschema

```diff
@@ -1,70 +1,240 @@
+                    "running",
+                    "SUCCEEDED",
+                    "DEAD",
+                    "Succeeded",
+                    "Dead",
+                    "succeeded",
+                    "dead"
+                  ]
                 }
+              },
+              "required": [
+                "task"
+              ],
+              "additionalProperties": false
+            },
+            "^(streq|strneq)(_.*)?$": {
+              "type": "object",
+              "properties": {
+                "left": {
+                  "type": "string"
+                },
+                "right": {
+                  "type": "string"
+                }
+              },
+              "required": [
+                "left",
+                "right"
+              ],
+              "additionalProperties": false
+            },
+            "^cycleexistdep(_.*)?$": {
+              "type": "object",
+              "properties": {
+                "cycle_offset": {
+                  "$ref": "#/$defs/TimePattern"
+                }
+              },
+              "additionalProperties": false
+            },
+            "^taskvalid(_.*)?$": {
+              "type": "object",
+              "properties": {
+                "task": {
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false
+            },
+            "^metataskdep(_.*)?$": {
+              "type": "object",
+              "properties": {
+                "attrs": {
+                  "type": "object",
+                  "properties": {
+                    "metatask": {
+                      "type": "string"
+                    },
+                    "cycle_offset": {
+                      "$ref": "#/$defs/TimePattern"
+                    },
+                    "state": {
+                      "type": "string",
+                      "enum": [
+                        "SUCCEEDED",
+                        "DEAD",
+                        "Succeeded",
+                        "Dead",
+                        "succeeded",
+                        "dead"
+                      ]
+                    },
+                    "threshold": {
+                      "type": "number",
+                      "minimum": 0,
+                      "maximum": 1
+                    }
+                  },
+                  "required": [
+                    "metatask"
+                  ],
+                  "additionalProperties": false
+                }
+              },
+              "additionalProperties": false
+            },
+            "^timedep(_.*)?$": {
+              "$ref": "#/$defs/TimePattern"
+            },
+            "^some(_.*)?$": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              },
+              "minItems": 1
             },
-            "required": ["attrs", "cycledefs", "log"]
+            "^threshold(_.*)?$": {
```

Comment:
It looks like `threshold` is an attribute of `some` in the Rocoto docs.

Reply:
Woah, that got shifted over somewhere along the way, let me just do a full doc recheck, thanks for catching this!

Reply:
Ah, it also has to loop too, I'll get that in